### PR TITLE
NEO Filipino Localization #3

### PR DESCRIPTION
**Pull request by burdagay**
For review

### CODE REVIEW

File: tl-tl/sc/getting-started-java.md [hunk 1]
```diff
@@ -2,98 +2,98 @@
...
+Ang mga smart na kontrata ay sinusulat gamit ang mga mataas na lebel na lengguwahe kagaya ng Java, C#, Python, Kotlin (at iba pa...) at kinukompayl para maging AVM (ang Virtual Machine bytecode ng Neo) para mapatakbo ito sa Neo network.
 
-We currently recommend C# for developing smart contracts. The Java compiler is still in development but the current version (neoj) can handle basic methods.
+Kami ay kasalukuyang nagrerekomenda na gumamit ng C# para gumawa ng mga smart na kontrata. Ang Java compiler ay ginagawa pa rin hanggang sa ngayon pero ang kasalukuyang bersyon (neoj) ay maari nang makapagtakbo ng mga basic na method.
 
-This section contains a tutorial that guides you in configuring the Java development environment for NEO smart contracts. It also gives you an idea of ​​how to create a smart contract project and how to compile it.
+Ang seksyon na ito ay naglalaman ng isang tutoryal na gagabay sa iyo sa pagkonpigyura ng Java development environment para sa mga NEO smart na kontrata. Ito rin ay magbibigay ng ideya kong papano gumawa ng isang smart na kontratang proyekto at paano ito i-kompayl.
```
Comment (josteinfrancois) [inline tl-tl/sc/getting-started-java.md]: ideya kung papaano

File: tl-tl/sc/getting-started-java.md [hunk 2]
```diff
@@ -2,98 +2,98 @@
...
-3. Build the neoj (C#) compiler that converts JVM code into AVM code (neoj.exe on Windows)
-4. Use neoj to compile your .class file (.avm)
-5. Download Neo's Node GUI to connect to Neo's Testnet network
-6. Deploy your .avm script to publish the smart contract to the network
-7. Invoke your .avm script to execute your smart contract
+Mga Paalala: Ang proseso ay naglalaman ng mga sumusunod na mga hakbang:
+1. Sumulat ng Java na code (.java) para sa mga class na umi-extend sa FunctionCode o VerificationCode na parte sa Neo Framework Library (JAR)
+2. Gumamit ng normal na Java compiler para i-kompayl ang code para maging Java bytecode (.class)
+3. I-build ang neoj (C#) na compiler na kino-convert ang JVM na code para maging AVM na code (neoj.exe sa Windows)
+4. Gamitin ang neoj para icompile ang iyong .class na file (.avm)
```
Comment (josteinfrancois) [inline tl-tl/sc/getting-started-java.md]: i-compile

File: tl-tl/sc/getting-started-java.md [hunk 3]
```diff
@@ -2,98 +2,98 @@
...
+5. I-downloand ang Node GUI ng Neo para kumonekta sa Testnet na network ng Neo.
+6. I-deploy ang iyong .avm na script para ipublish ang smart na kontrata sa network
```
Comment (josteinfrancois) [inline tl-tl/sc/getting-started-java.md]: i-publish

File: tl-tl/sc/getting-started-java.md [hunk 4]
```diff
@@ -2,98 +2,98 @@
...
+7. I-invoke ang iyong .avm na script para patakbuhin ang iyong smart na kontrata
 
-### Detailed Instructions
+### Mga Detalyadong Instruksyon
 
-## Tools
+## Mga Tool
 
-The most efficient way of getting these steps done is to download and compile all the tools you will need:
+Ang pinakaepisyenteng pamamaraan para magawa ang mga sumusunod na mga hakbang ay ang pagdownload at pagkompayl ng lahat ng tools na kakailanganin mo:
```
Comment (josteinfrancois) [inline tl-tl/sc/getting-started-java.md]: you could translate `tools` to `kasangkapan` or `kagamitan`

File: tl-tl/sc/getting-started-java.md [hunk 5]
```diff
@@ -2,98 +2,98 @@
...
-3. Download an IDE for Java (optional but recommended), e.g. IntelliJ or Eclipse.
-4. Download an IDE for C# - currently the neoj compiler needs to be built manually as it is not in wide distribution release format. Recommended is to get Visual Studio 2017 which is free.
+1. I-download ang Node GUI ng Neo. Sa pagkakasulat nito, nirerekomenda na gumamit ka ng BETA developer GUI dahil mayroon itong mga ekstrang debugging feature na nagpapadali ng lahat. [CoZ NEO GUI](https://github.com/CityOfZion/neo-gui-developer). Ito ay mayroong mga default preset sa Testnet at kinakailangan mong maghintay (hanggang sa ilang oras) para ito ay mag-sync ng lubos.
+2. I-download ang Neo Framework Library JAR. Nandito ang kasalukuyang pinakabago na bersyon: [org.neo.smartcontract.framework JAR](https://github.com/CityOfZion/neo-java-sdk/blob/master/target/org.neo.smartcontract.framework.jar)
+3. Magdownload ng isang IDE para sa Java (opsyonal ito pero rinrekomenda), e.g. IntelliJ o Eclipse.
```
Comment (josteinfrancois) [inline tl-tl/sc/getting-started-java.md]: - `Mag-download` separate foreign word with dash
- `inirerekomenda`

File: tl-tl/sc/getting-started-java.md [hunk 6]
```diff
@@ -2,98 +2,98 @@
...
+4. Magdownload ng isang IDE para sa C# - sa kasalukuyan, ang neoj compiler ay kinakailangang likhain ng mano-mano dahil ito ay hindi naka wide distribution release na format.
 
-## Development Tools
+## Mga Development Tool
 
 ### 1. Visual Studio 2017
 
-If you have already installed Visual Studio 2017 on your computer and checked for .NET Cross-Platform Development at the time of installation, you can skip this section.
+Kong ikaw ay naka-install na ng Visual Studio 2017 sa iyong kompyuter at naka-check sa .NET Cross-Platform Development sa panahon na nag-iinstall, pwede mong laktawan ang seksyon na ito.
```
Comment (josteinfrancois) [inline tl-tl/sc/getting-started-java.md:37]: `Kung ikaw`

File: tl-tl/sc/getting-started-java.md [hunk 7]
```diff
@@ -2,98 +2,98 @@
...
 
-Download and install:
+Pag-download at pag-install:
 
-[Visual Studio download address](https://www.visualstudio.com/products/visual-studio-community-vs)
+[Ang address para i-download ang Visual Studio](https://www.visualstudio.com/products/visual-studio-community-vs)
 
-The installation process is very simple, follow the operation prompts step-by-step, it should be noted that you need to check the installation of `.NET Core cross-platform development`, otherwise you will not be able to open neo-vm project in step #3. The installation takes about ten minutes or up to an hour.
+Napakasimple lang ng proseso sa pag-install, sundin lang ang bawat hakbang sa mga operation prompt, dapat mong tandaan na kinakailangan mong e-check ang pag-install ng `.NET Core cross-platform development`, dahil kong hindi, hindi mo maaring buksan ang neo-vm na proyekto na nasa ikatlong hakbang. Ang pag-install ay tumatagal ng halos sampung minuto o hanggang sa isang oras.
```
Comment (josteinfrancois) [inline tl-tl/sc/getting-started-java.md]: - you could translate `check` to `suriin` or `i-check`
- `dahil kung hindi, hindi mo maaaring`

File: tl-tl/sc/getting-started-java.md [hunk 8]
```diff
@@ -2,98 +2,98 @@
...
+Kinakailangan na natin ngayong idagdag ang direktoryong ito sa ating execution path. Ang PATH ay isang system variable na ginagamit ng iyong operating system para matagpuan ang mga kinakailangang mga executable galing sa command line o Terminal window.
 
-**Windows 10 and Windows 8:**
+**Windows 10 at Windows 8:**
 
-  In Search, search for and then select: System (Control Panel)
-  Click the Advanced system settings link.
-  Click Environment Variables. In the section System Variables, find the PATH environment variable and select it. Click Edit. If the PATH environment variable does not exist, click New.
-  In the Edit System Variable (or New System Variable) window, specify the value of the PATH environment variable. Click OK. Close all remaining windows by clicking OK.
+  Sa Search, maghanap ng at pagkatapos pumili: System (Control Panel)
```
Comment (josteinfrancois) [inline tl-tl/sc/getting-started-java.md:65]: at pagkatapos ay piliin ang:

File: tl-tl/sc/getting-started-java.md [hunk 9]
```diff
@@ -2,98 +2,98 @@
...
+  Piliin ang Properties galing sa context menu.
+  I-click ang Advanced system settings na link.
+  I-click ang Environment Variables. Sa seskyon na System variables, hanapin ang PATH environment variable at piliin ito. I-click ang Edit. Kung wala ang PATH environment variable, i-click ang New.
+  Sa Edit System Variable (o New System Variable) na window, itukoy ang value ng PATH environment variable. I-click ang OK. Isara ang lahat ng natitirang mga window sa pamamagitan ng pag-click ng OK.
 
-![edit environmental variables](/assets/edit_environmental_variables.png)
+![i-edit ang mga environmental variable](/assets/edit_environmental_variables.png)
 
-Now run Command or PowerShell, and enter neoj.exe. If there is no error and the output shows the version number (as shown) the environment variable configuration is successful
+Ngayon patakbuhin ang Command o Powershell, at i-enter ang neoj.exe. Kung wala error at ang output ay nagpapakita ng numero ng bersyon (kagyaa ng pinapakita) ang pag-konpigyura ng environment variable ay naging matagumpay.
```
Comment (josteinfrancois) [inline tl-tl/sc/getting-started-java.md:80]: - `Kung walang error`
- `kagaya ng pinapakita`

File: tl-tl/sc/getting-started-java.md [hunk 10]
```diff
@@ -2,98 +2,98 @@
...
+TANDAAN. Ang mga gumagamit ng Windows 7 SP1 ay maaring maka-enkwentro ng error na "Unhandled Exception: System.DllNotFoundException: Unable to load DLL 'api-ms-win-core-console-l2-1-0.dll': The specified module could not be found". Ang kinakailangang 'api-ms-win-core-console-l2-1-0.dll' na file ay matatagpuan lamang sa Windows 8 o mas bago pa na mga bersyon. Ang error na ito ay maaring maresolba sa pamamagitan ng pagkuha na kopya ng 'api-ms-win-core-console-l2-1-0.dll' at paglagay nito sa direktoryo na C:\Windows\System32. Ang dll ay maaring makita sa maraming mga lugar sa iyong sistema (maghanap sa iyong kompyuter at kopyahi o i-paste ito sa loob ng \System32), pero pwede rin itong hanapin online.
```
Comment (josteinfrancois) [inline tl-tl/sc/getting-started-java.md:85]: - `maaaring`

File: tl-tl/sc/getting-started-java.md [hunk 11]
```diff
@@ -102,23 +102,22 @@ import org.neo.smartcontract.framework.services.neo.Storage;
...
-        return Storage.get(Storage.currentContext(),"Greeting to the World");
+        Storage.put(Storage.currentContext(), "Pagbati para sa Mundo", "Hello Mundo!");
+        return Storage.get(Storage.currentContext(),"Pagbati para sa Mundo");
     }
 
 }
 ```
 
-Build the project which will give you `HelloWorld.class` in your out folder.
+I-build ang proyekto na kung saan ay magbibigay sa yo ng `HelloWorld.class` sa iyong out na folder.
```
Comment (josteinfrancois) [inline tl-tl/sc/getting-started-java.md]: ``sa'yo``

File: tl-tl/sc/getting-started-java.md [hunk 12]
```diff
@@ -102,23 +102,22 @@ import org.neo.smartcontract.framework.services.neo.Storage;
...
 
-Build the project which will give you `HelloWorld.class` in your out folder.
+I-build ang proyekto na kung saan ay magbibigay sa yo ng `HelloWorld.class` sa iyong out na folder. 
 
-Then using neoj, run cmd.exe and execute:
+Ngayon habang ginagamit ang neoj, patakbuhin ang cmd.exe at i-execute ang:
 > neoj.exe HelloWorld.class
 
-If successful, it will create HelloWorld.avm which you can now use as smart contract bytecode.
+Kung matagumpay, ito ay gagawa ng HelloWorld.avm na kung saan ikaw ay maarin nang gumamit ng bytecode na smart na kontrata.
```
Comment (josteinfrancois) [inline tl-tl/sc/getting-started-java.md]: `maaari nang`

File: tl-tl/sc/getting-started-python.md [hunk 13]
```diff
@@ -1,27 +1,27 @@
-# Getting Started with neo-python
+# Pagsisimula gamit ang neo-python
 
 *Main author: [@localhuman](https://github.com/localhuman)*
 
-neo-python, the python node and SDK for the NEO blockchain, enables you to build, test, and run smart contracts on the Neo blockchain. It provides the following main functionalities:
+Ang neo-python, ang python node at SDK para sa NEO blockchain, ay nagbibigay sa iyo ng kakayahan na mag-build, mag-test, at magpatakbo ng mga smart na kontrata sa Neo blockchain. Ito ay nagbibigay ng mga sumusunod na pangunahing functionalidad:
```
Comment (josteinfrancois) [inline tl-tl/sc/getting-started-python.md]: you could just retain the word `functionality` here

File: tl-tl/sc/getting-started-python.md [hunk 14]
```diff
@@ -1,27 +1,27 @@
...
+- [NEP2](https://github.com/neo-project/proposals/blob/master/nep-2.mediawiki) at [NEP5](https://github.com/neo-project/proposals/blob/master/nep-5.mediawiki) compliant wallet na functionalidad
 - RPC Client
-- `Runtime.Log` and `Runtime.Notify` event monitoring
+- Pagmonitor sa `Runtime.Log` at `Runtime.Notify` na event
 
-## Installing the Python library
+## Pag-install ng Python library
 
-You need to install the libleveldb library. Install [Python 3.5](https://www.python.org/downloads/release/python-354/) to make sure you don't run into any issues with your version of Python being different than the current maintainer's version. Note that Python 3.6 is not currently supported due to incompatibilities with the byteplay module.
+Kinakailangan mong i-install ang libleveldb na library. I-install ang [Python 3.5](https://www.python.org/downloads/release/python-354/) para masiguradong walang isyu na magaganap sa iyong ibang bersyon ng Python na di kapareho sa bersyon ng maintainer. Tandaan na ang Python 3.6 ay hindi sinusuportahan dahil sa mga incompatibility na mga isyu sabyteplay na module.
```
Comment (josteinfrancois) [inline tl-tl/sc/getting-started-python.md]: `sa byteplay`

**Review by josteinfrancois — COMMENTED** [review]
Hey, @burdagay. I have reviewed your work. Good job, I only have a few minor suggestions. Thanks.

File: tl-tl/sc/getting-started-java.md (hunk 1 above)
Comment (burdagay) [reply tl-tl/sc/getting-started-java.md]: fix provided

File: tl-tl/sc/getting-started-java.md (hunk 2 above)
Comment (burdagay) [reply tl-tl/sc/getting-started-java.md]: fix provided

File: tl-tl/sc/getting-started-java.md (hunk 3 above)
Comment (burdagay) [reply tl-tl/sc/getting-started-java.md]: fix provided

File: tl-tl/sc/getting-started-java.md (hunk 4 above)
Comment (burdagay) [reply tl-tl/sc/getting-started-java.md]: Changed `tool` to `kagamitan`

File: tl-tl/sc/getting-started-java.md (hunk 5 above)
Comment (burdagay) [reply tl-tl/sc/getting-started-java.md]: fix provided

File: tl-tl/sc/getting-started-java.md (hunk 10 above)
Comment (burdagay) [reply tl-tl/sc/getting-started-java.md:85]: fix provided

File: tl-tl/sc/getting-started-java.md (hunk 11 above)
Comment (burdagay) [reply tl-tl/sc/getting-started-java.md]: fix provided

File: tl-tl/sc/getting-started-java.md (hunk 12 above)
Comment (burdagay) [reply tl-tl/sc/getting-started-java.md]: fix provided

File: tl-tl/sc/getting-started-python.md (hunk 13 above)
Comment (burdagay) [reply tl-tl/sc/getting-started-python.md]: fix provided

File: tl-tl/sc/getting-started-python.md (hunk 14 above)
Comment (burdagay) [reply tl-tl/sc/getting-started-python.md]: fix provided

**Pull request merged**